src/ResourceManager/Profile/Commands.Profile/StartupScripts/AzError.ps1

```diff
@@ -0,0 +1,67 @@
+﻿function Write-InstallationCheckToFile
+{
+    Param($installationchecks)
+    if (Get-Module AzureRM.Profile -ListAvailable)
+    {
+        Write-Warning "Both Az and AzureRM modules were detected on your machine. Az and AzureRM module cannot be run side-by-side, please run 'Uninstall-AzureRm' to remove all AzureRm modules from your machine. More information can be found here: aka.ms/azps-migration-guide"
+    }
+
+    $installationchecks.Add("AzSideBySideCheck","true")
+    try
+    {
+        Remove-Item -Path $pathToInstallationChecks
```

Comment:
If you condition this removal on Test-Path, I think you can make this call in all 3 cases below.